overlay/db/locality.py

```diff
@@ -22,7 +23,14 @@ def discover_gis_joins() -> dict:
     # Connect to local mongod instance; connecting to mongos instance will find all GISJOINs in entire cluster,
     # rather than just the local shards.
     gis_join_counts: dict = {}  # { gis_join -> count }
-    client: MongoClient = MongoClient("mongodb://localhost:27017")
+    info("Inside locality.py, just above the call to mongod")
+    info("Inside locality.py, password and username: ", password, username)
```

Comment:
We should probably make these DEBUG logs. But this is okay for now.

Reply:
Yeppers. This PR is messy, it's definitely just to update main with the streaming and auth so we don't have further issues. I intend to clean it up in a future PR. I just wanted you to have to updates.